examples/pf.jl

```diff
@@ -12,5 +12,8 @@ function pf(datafile, npartition=2, solver="default")
     println(solver)
     
     pf = PowerSystem.PowerNetwork(datafile)
-    return solve(pf, npartition, solver);
+    x = ExaPF.PowerSystem.get_x(pf)
+    u = ExaPF.PowerSystem.get_u(pf)
+    p = ExaPF.PowerSystem.get_p(pf)
```

Comment:
Nice! Why we do not use multiple dispatch at the end? (I think both solutions are valid, though)

Reply:
I had some questions about it but since I did it on Saturday I didnt want to bother anyone. Can we talk about it in our next meeting? Ill push it like this for now and later I can change it to the multiple dispatch version.